kong/db/dao/init.lua

```diff
@@ -221,6 +221,11 @@ function _M.new(db, schema, strategy, errors)
 end
 
 
+function DAO:truncate()
```

Comment:
I think this function is never used. I am not sure it belongs in the DAO level, either - maybe it belongs on the "instantiated strategy" (combination of schema + strategy) level.